code/_helpers/delays.dm

```diff
@@ -0,0 +1,32 @@
+//////////////////////////////////
+// /vg/ MODULARIZED DELAYS - by N3X15
+//////////////////////////////////
```

Comment:
what is a delay controller? What's it for?

Reply:
Basically just a thin wrapper to remove duplication with `world.time` based cooldowns. You can set a delay and then check later whether the controller is still blocked.